mainnet/docker-compose.yml

```diff
@@ -0,0 +1,12 @@
+version: "3.9"
+services:
+  itchysats-mainnet:
+    image: ghcr.io/itchysats/itchysats/taker:0.4.19
+    restart: on-failure
+    ports:
+      - 8000:8000
+    volumes:
+      - ./data:/data
+    command: |
+      --log-level=Info
+      mainnet
```

Comment:
This is the default command. It does not have to be specified, but we can :)

Reply:
I know - left it there to be explicit (might at some point try a different trick, moving it to some `env` config if the user will be specifying password there at some point)